<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatterPlotDialog</class>
 <widget class="QDialog" name="ScatterPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>591</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>591</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X Axis</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="x_model_combo_box"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Element</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="x_element_combo_box"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ens</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="x_ens_spin_box">
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_min_spin_box">
        <property name="minimum">
         <double>-200000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_max_spin_box">
        <property name="minimum">
         <double>-200000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y Axis</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="y_model_combo_box"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Element</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="y_element_combo_box"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Ens</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="y_ens_spin_box">
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="y_min_spin_box">
        <property name="minimum">
         <double>-200000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="y_max_spin_box">
        <property name="minimum">
         <double>-200000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ScatterPlotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ScatterPlotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
